rune/libenclave/internal/runtime/pal/skeleton/sgxutils.c

```diff
@@ -49,44 +49,34 @@ static inline uint64_t get_xcr0()
 	return xgetbv(0);
 }
 
-static bool try_get_xcr0(uint64_t *value)
+static uint64_t try_get_xcr0()
 {
 	int cpu_info[4] = {0, 0, 0, 0};
 
-	*value = SGX_XFRM_LEGACY;
-
 	// check if xgetbv instruction is supported
 	__cpuid(cpu_info, 1);
 	// ecx[27:26] indicate whether support xsave/xrstor, and whether enable xgetbv, xsetbv
 	if (!(cpu_info[2] & (1<<XSAVE_SHIFT)) || !(cpu_info[2] & (1<<OSXSAVE_SHIFT)))
```

Comment:
Remove `!(cpu_info[2] & (1<<XSAVE_SHIFT)) ||`, because if save is unavailable, it is impossible to set cr4.OSXSAVE.